releases/release-1.32/README.md

```diff
@@ -0,0 +1,89 @@
+| Docs deadline — PRs ready for review                                                   | Docs Lead                     | Tuesday 12th November 2024                                                    | week 10 |                                                        |
+| Deprecations and Removals blog published                                               | Comms                         | Thursday 14th November 2024                                                   | week 10 |                                                        |
+| **Begin [Burndown]** (Monday, Wednesday, and Friday meetings)                          | Lead                          | Monday 18th November 2024                                                     | week 11 |                                                        |
+| Release Highlights deadline                                                                  | Comms                         | Tuesday 19th November 2024                                                    | week 11 |                                                        |
+| Start final draft of Release Notes                                                     | Release Notes Lead            | Tuesday 19th November2024                                                     | week 11 |                                                        |
+| **Begin [Code Freeze]**                                                                | Branch Manager                | 02:00 UTC Wednesday 20th November 2024 / 19:00 PDT Tuesday 19th November 2024 | week 11 |                                                        |
+| Release blog ready to review                                                           | Comms / Docs                  | 02:00 UTC Wednesday 20th November 2024 / 19:00 PDT Tuesday 19th November 2024 | week 11 |                                                        |
+| Feature blogs ready to review                                                          | Enhancement Owner / SIG Leads | Friday 22nd November 2024                                                     | week 11 |                                                        |
+| Burndown Meetings daily (Tuesday & Thursday over Slack)                                | Lead                          | Monday 25th November 2024                                                     | week 12 |                                                        |
+| [**Test Freeze**]                                                                      | Branch Manager                | 01:00 UTC Wednesday 27th November 2024 / 19:00 PDT Tuesday 26th November 2024 | week 12 |                                                        |
+| release-1.32 branch created                                                            | Branch Manager                | Tuesday 26th November 2024                                                    | week 12 |                                                        |
```

Comment:
This is also a holiday week in the US, so we should work early to make sure we have review coverage outside the US in terms of release managers.

Reply:
Thanksgiving week is 25-29, the problem I see with delaying things is that it could move the release date to week 15, which is closer to Christmas (also typically a period where vacations/holidays are a concern), but it's doable e.g.:

- Move release date to Dec 18th
- Move Docs Freeze to Dec 3rd
- Move `rc-1` to Dec 10 
 
... and adjust the rest, giving more days for the "ready to review"  ones. 

Or we could just keep it as is and prepare for it, as you mentioned @jeremyrickard .

Reply:
Understood, will make sure that that is properly relayed to everyone.